features/issues/components/issue-list/issue-filter.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components";
+import { Select, Input, StyledButton } from "@features/ui";
+import { SelectState } from "@features/ui/select/select";
+import { InputIcon, InputLabel, InputState } from "@features/ui/input/input";
+import { ButtonIcon } from "@features/ui/button/button";
+import { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 25px;
+`;
+
+const FilterContainer = styled.div`
+  display: flex;
+`;
+
+export const IssueFilter = () => {
+  const [statusSelectOpen, setStatusSelectOpen] = useState(false);
+  const [levelSelectOpen, setLevelSelectOpen] = useState(false);
+  const [currentState, setCurrentState] = useState("All status");
+  const [currentLevel, setCurrentLevel] = useState("All level");
+  const stateSelectOptions = ["All status", "Unresolved", "Resolved"];
+  const levelSelectOptions = ["All level", "Error", "Warning", "Info"];
+  const issues = useSelector((state: any) => state.issues);
```

Comment:
No need for this, you could simply use the `useGetIssues` hook here as well (same as in the `issue-list` component). That makes Redux overall obsolete